cli/src/program_v4.rs

```diff
@@ -322,23 +330,21 @@ impl ProgramV4SubCommands for App<'_, '_> {
                         )),
                 )
                 .subcommand(
-                    SubCommand::with_name("dump")
-                        .about("Write the program data to a file")
+                    SubCommand::with_name("download")
+                        .about("Download the executable of a program to a file")
                         .arg(
-                            Arg::with_name("account")
```

Comment:
Here we are switching the order of the positional arguments. Is it necessary? i.e. the positional arguments, and switching their ordering.

Reply:
Same here for consistency with `deploy`: The path to ELF argument is positional and the `--program-id` is named now.